server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolParser.g4

```diff
@@ -1073,13 +1074,39 @@ cancelCall
 // change priority statement
 
 changePriorityStatement
-    : CHANGE PRIORITY TO? (changePriorityLiteral | generalIdentifier)
+    : CHANGE PRIORITY TO? (numericLiteral | generalIdentifier)
     ;
 
 changePriorityLiteral
```

Comment:
We might get rid of this.

Reply:
The references to changePriorityLiteral and priorityLiteral have been removed, but I still left the items. I can remove them and re-submit the PR, or remove them in my future PR, whichever you prefer.

Reply:
Doesn't impact. I guess we can get rid of these in future PR.